addons/web/static/src/core/errors/error_dialog.scss

```diff
@@ -6,7 +6,7 @@
             overflow: auto;
 
             pre {
-                overflow: initial;
+                overflow: auto;
```

Comment:
Hello :wave: 
we can remove it :smile: 
I think I need to see this `I checked the styling of the error dialog using the QUnit test base_automation. It might not load the same assets then the Odoo backend view` :eyes:

Reply:
Hello @julien-banken 
are you talking about this test? 
I tried and ran the Qunit test and the error dialog layout is fine or am I misunderstanding :thinking: 
![image](https://github.com/odoo/odoo/assets/121081839/eb16a29d-f32c-4b91-9940-ddee39a62546)

Reply:
Hello @thsh-odoo

Yes, I ran that QUnit test in debug mode to check the dialog styling. I used that test because I don't know a flow in Odoo to trigger an error displaying the dialog we're updating. I'm not really familiar with the sales module.

I said that we need to check if the rule is necessary because the stylesheet loaded in the test suites might be different than the stylesheet loaded in the Odoo backend. So, it could be possible that the rule I'm talking about is not defined in the backend or that another rule is overwriting that rule.